### DISCUSSION TITLE
View source code

### DISCUSSION THREAD

**Discussion by maxdiebold** [davidgranstrom/scnvim#199] [Q&A] [answered]
Pardon me if I simply missed it, but is there a simple way to view the source code for a class from Nvim? When I click the "Source" link in the docs, I receive the error "Message 'implementationClass' not understood", and regardless it would be handy to navigate to the source from Nvim with a key command.

**Comment by davidgranstrom** [comment, marked as answer]
Are you using the HelpBrowser (GUI) or the built-in help inside nvim? 

Viewing the definition of a keyword is otherwise a built-in feature of nvim once you have generated a `tags` file. Starting scnvim and running the command `:SCNvimGenerateAssets` will do this for you. After this you can jump to the implementation for the word under the cursor with `C-]` and then use `C-o` to jump back to where you were. Also take a look at `gC-]` if there are multiple definitions of the keyword (often the case with methods). Also see `:h CTRL-]` for a detailed explanation.

**Reply by maxdiebold** [reply]
Thanks, works like a charm! I'd actually just run that command the day prior, but I was still a bit foggy on tags.
